netbox/templates/dcim/device.html

```diff
@@ -179,8 +179,8 @@ <h5 class="card-header">
                                 <a href="{{ object.primary_ip4.get_absolute_url }}">{{ object.primary_ip4.address.ip }}</a>
                                 {% if object.primary_ip4.nat_inside %}
                                   (NAT for {{ object.primary_ip4.nat_inside.address.ip|linkify }})
-                                {% elif object.primary_ip4.nat_outside %}
-                                  (NAT: {{ object.primary_ip4.nat_outside.address.ip|linkify }})
+                                {% elif object.primary_ip4.nat_outside.count > 0 %}
```

Comment:
When checking whether an object exists, it's best practice to use `.exists()` rather than `.count()` as it's slightly more efficient and better conveys intent.

```suggestion
                                {% elif object.primary_ip4.nat_outside.exists %}
```